tools/development/helpers/format-cpp.sh

```
@@ -5,8 +5,7 @@
 project_directory="$(git rev-parse --show-toplevel)"
 
 pushd "${project_directory}" > /dev/null
-
-    clang-format -i -style=file:thirdparty/clang/.clang-format $(find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp' -o -name '*.tpp') \
+    clang-format -i -style=file:thirdparty/clang/.clang-format $(find ~+/src/ ~+/include/ ~+/test/ ~+/bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp' -o -name '*.tpp') \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)